src/components/checkboxFilter/index.tsx

```diff
@@ -36,6 +35,7 @@ type Props<ItemKey extends string> = {
    * @param newState        contains the new state of the whole filter group
    */
   onApply: (updatedItem: Item<ItemKey>, newState: State<ItemKey>) => void;
+  numberOfColumns?: number;
```

Comment:
to avoid misunderstandings you might want to go with:

```suggestion
  numberOfColumnsOnDesktop?: number;
```